index.js

```diff
@@ -2,6 +2,21 @@ const ham = document.getElementById('ham');
 
 const div = document.querySelector('#nav3');
 
+const form = document.querySelector('#form');
+
+const email = document.querySelector('#mail');
+
+const errorElement = document.getElementById('error');
+
+form.addEventListener('submit', (e) => {
+  if (email.value
+    !== email.value.toLowerCase()) {
+    e.preventDefault();
+    errorElement.style.display = 'block';
+    errorElement.textContent = 'invalid email, please use lowerCase';
```

Comment:
- [ ] I kindly suggest that you ensure that your error message shows up close to the submit button
![image](https://user-images.githubusercontent.com/52934128/175769458-fc806155-7685-4311-bae5-0aaa9629b587.png)